### PR TITLE
docs: upgrade 0.8.3

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Review by lumianph — APPROVED** [review]

**Comment by lumianph** [comment]
let's merge #3498 as well

File: release/conf/openmldb-env.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env bash
-export OPENMLDB_VERSION=0.8.2
+export OPENMLDB_VERSION=0.8.3
 # openmldb mode: standalone / cluster
 export OPENMLDB_MODE=${OPENMLDB_MODE:=standalone}
```
Comment (vagetablechicken) [inline release/conf/openmldb-env.sh:4]: Shall we set OPENMLDB_MODE default to cluster? maybe next pr.
It's difficult to use sbin/stop-tablets.sh or other scripts, we need to set OPENMLDB_MODE to env manually, or OPENMLDB_MODE=cluster sbin/xxx.sh

**Review by vagetablechicken — APPROVED** [review]

File: release/conf/openmldb-env.sh (hunk 1 above)
Comment (dl239) [reply release/conf/openmldb-env.sh:4]: yes, in most  case we deploy with cluster mode.

Comment (lumianph) [reply release/conf/openmldb-env.sh:4]: +1

Comment (dl239) [reply release/conf/openmldb-env.sh:4]: #3511

**Pull request merged**